cpp/src/parquet/column_reader.h

```diff
@@ -115,11 +116,26 @@ class PARQUET_EXPORT PageReader {
                                           bool always_compressed = false,
                                           const CryptoContext* ctx = NULLPTR);
 
+  // If skip_page_callback_ is set, NextPage() must use this callback to determine if it
+  // should return or skip and move to the next page. If the callback function returns
```

Comment:
we might want to clarify the contract here further:
1.  Pages will be called in the order they appear in the column?
2. The callback will be called at most once per page?

Reply:
3.  set_skip_page_callback is expected to be called at most once?

Reply:
Done